src/templates/ja/2020/methodology.html

```diff
@@ -0,0 +1,426 @@
+      <h3 id="brainstorming"><a href="#brainstorming" class="anchor-link">ブレインストーミング</a></h3>
+
+      <p>
+        The 2020 Web Almanac kicked off in June 2020 as a <a href="https://twitter.com/rick_viscomi/status/1273135952848977920?s=20">request for new authors</a>. After a round of brainstorming, 22 chapters were solidified and we began <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues/876">assigning subject matter experts and peer reviewers to each chapter</a>. This process had some inherent bias because of the challenge of getting volunteers to commit to a project of this scale. Thus, many of the contributors are members of the same professional circles. One explicit goal for future editions of the Web Almanac is to encourage even more inclusion of underrepresented and heterogeneous voices as authors and peer reviewers.
+      </p>
+
+      <p>
+        We spent July 2020 pairing people with chapters and getting their input to finalize the individual metrics that will make up each chapter.
+      </p>
+    </section>
+
+    <section>
+      <h3 id="analysis"><a href="#analysis" class="anchor-link">Analysis</a></h3>
+
+      <p>
+        In July and August 2020, with the stable list of metrics and chapters, data analysts triaged the metrics for feasibility. In some cases, <a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">custom metrics</a> needed to be created to fill gaps in our analytic capabilities.
+      </p>
+
+      <p>
+        Throughout August 2020, the HTTP Archive data pipeline crawled several million websites, gathering the metadata to be used in the Web Almanac.
+      </p>
+
+      <p>
+        The data analysts began writing queries to extract the results for each metric. In total, <strong>TBC queries</strong> were written by hand! You can browse all of the queries by chapter in the <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">sql/2020</a> directory of the project's GitHub repository.
+      </p>
+    </section>-->
+
+    <section>
+      <h3 id="interpretation"><a href="#interpretation" class="anchor-link">解釈</a></h3>
+
+      <p>
+        著者はアナリストと協力して、結果を正しく解釈し、適切な結論を導き出しました。著者はそれぞれの章を執筆する際には、これらの統計からウェブの状態についての枠組みを支持しました。レビュワーは著者と協力して、分析の技術的な正確さを確認しました。
+      </p>
+
+      <p>
+        読者に結果をよりわかりやすく伝えるために、ウェブ開発者やアナリストは、この章に埋め込むデータの可視化を作成しました。いくつかの可視化は、結論を把握しやすくするために簡略化されています。たとえば、分布の完全なヒストグラムを表示するのではなく、ほんの一握りのパーセンタイルだけを表示しています。特に断りのない限り、すべての分布は、平均ではなくパーセンタイル、特に中央値（50％）を使用して要約されています。
+      </p>
+
+      <p>
+        最後に、編集者は各章を修正し、簡単な文法的な誤りを修正し、読書体験全体に一貫性を持たせるようにしました。
+      </p>
+    </section>
+
+    <section>
+      <h2 id="looking-ahead"><a href="#looking-ahead" class="anchor-link">将来を見据えて</a></h2>
+
+      <p>
+        Webアルマナックの2020年版は、内省と前向きな変化への取り組みというWebコミュニティの毎年恒例の伝統の2番目です。ここまで来れたのは、多くの献身的な<a href="{{ url_for('contributors', year=year, lang=lang) }}">コントリビューター</a>のおかげで、記念碑的な努力をしてきました。
```

Comment:
This line could do with a particular review

Reply:
```suggestion
        Web Almanacの2020年版は毎年恒例、Webコミュニティ伝統の2回目で、内省と前向きな変化への取り組みです。ここまで来れたのは、多くの献身的な<a href="{{ url_for('contributors', year=year, lang=lang) }}">コントリビューター</a>のおかげで、記念碑的な努力をしてきました。
```